lib/atomic_cache/atomic_cache_client.rb

```diff
@@ -37,8 +37,7 @@ def initialize(storage: nil, timestamp_manager: nil, default_options: {}, logger
     # @option options [Numeric] :max_retries (5) Max times to rety in waiting case
     # @option options [Numeric] :backoff_duration_ms (50) Duration in ms to wait between retries
     # @yield Generates a new value when cache is expired
-    def fetch(keyspace, options=nil)
-      options ||= {}
+    def fetch(keyspace, options={})
```

Comment:
All options input from the user was changed from `options=nil` to `options={}`.  In the case where the user passes in a `nil`, that's an error case and an exception will be encountered.  As it was, by doing `=nil` then `||= {}` it was silently hiding error cases.  If something is passed in as options, it needs to be hash-like.